librz/bin/format/java/class_bin.c

```diff
@@ -401,7 +401,10 @@ RZ_API char *rz_bin_java_class_name(RzBinJavaClass *bin) {
 		return strdup("unknown_class");
 	}
 
-	return java_class_constant_pool_stringify_at(bin, index);
+	char *tmp = java_class_constant_pool_stringify_at(bin, index);
+	char *class_name = rz_str_newf("L%s;", tmp);
+	free(tmp);
+	return class_name;
 }
 
 RZ_API char *rz_bin_java_class_super(RzBinJavaClass *bin) {
```

Comment:
While you are on it, could you please also add an ownership mark to this function return value?